src/Cogs/doam.py

```diff
@@ -0,0 +1,126 @@
+import configparser
+import discord
+import src.Cogs.player as p
+from discord.ext import commands
+
+config_ini = configparser.ConfigParser()
+config_ini.read('config.ini')
+doam_channel_id = int(config_ini['Doam']['doam_channel_id'])
+doam_admin_role = int(config_ini['Doam']['doam_admin_role'])
+doam_fans_role = int(config_ini['Doam']['doam_fans_role'])
+
+
+class Doam(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(brief='',
+                      description='',
+                      aliases=['doam'])
+    async def dildoam(self, ctx):
+        doam_channel = self.bot.get_channel(doam_channel_id)
+
+        def get_player(msg):
+            return msg.author == ctx.author and msg.channel == doam_channel and msg.mentions
+
+        await doam_channel.send('**Team 1** (pitching first) \nPing the pitcher')
+        player = await self.bot.wait_for('message', check=get_player)
+        pitcher1 = player.mentions[0]
+
+        await doam_channel.send('**Team 1** (pitching first) \nPing the batter')
+        player = await self.bot.wait_for('message', check=get_player)
+        batter1 = player.mentions[0]
+
+        await doam_channel.send('**Team 2** (batting first) \nPing the pitcher')
+        player = await self.bot.wait_for('message', check=get_player)
+        pitcher2 = player.mentions[0]
+
+        await doam_channel.send('**Team 2** (batting first) \nPing the batter')
+        player = await self.bot.wait_for('message', check=get_player)
+        batter2 = player.mentions[0]
+
+        await doam_channel.send(f'<@&{doam_fans_role}> ITS DOAM TIME!')
+        team1_hrs = 0
+        team2_hrs = 0
+        rounds = 10
```

Comment:
Minor thing, but maybe this should be a class instead of a local variable, with the default of 10, in case the number of rounds changes in the future

Reply:
I ended up using it as a counter in sudden death which... might not be best coding practice but making a second variable for that just seems unnecessary. Just gonna leave it as is.